crm/modules/member/form.inc.php

```diff
@@ -27,12 +27,6 @@
 */
 function member_add_form () {
     
-    // Ensure user is allowed to add members
-    if (!user_access('member_add')) {
```

Comment:
Why was this removed?

Reply:
this is to allow the form to be rendered without anyone being logged in, so it can be used as part of the register form